Upstream insists on generating SSL testing certificates with relatively short
lifespan, which has repeatedly caused problems (ie, one day the regression
tests suddenly stop working).  Replace them with certificates with 20-year
lifespan.  We should periodically regenerate these, too, but at least not
very often.


diff -Naur mysql-5.0.95.orig/mysql-test/r/openssl_1.result mysql-5.0.95/mysql-test/r/openssl_1.result
--- mysql-5.0.95.orig/mysql-test/r/openssl_1.result	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/r/openssl_1.result	2012-02-03 13:50:57.830922123 -0500
@@ -3,8 +3,8 @@
 insert into t1 values (5);
 grant select on test.* to ssl_user1@localhost require SSL;
 grant select on test.* to ssl_user2@localhost require cipher "DHE-RSA-AES256-SHA";
-grant select on test.* to ssl_user3@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB";
-grant select on test.* to ssl_user4@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB" ISSUER "/C=SE/ST=Uppsala/L=Uppsala/O=MySQL AB";
+grant select on test.* to ssl_user3@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB/emailAddress=abstract.mysql.developer@mysql.com";
+grant select on test.* to ssl_user4@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB/emailAddress=abstract.mysql.developer@mysql.com" ISSUER "/C=SE/ST=Uppsala/L=Uppsala/O=MySQL AB";
 grant select on test.* to ssl_user5@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "xxx";
 flush privileges;
 connect(localhost,ssl_user5,,test,MASTER_PORT,MASTER_SOCKET);
diff -Naur mysql-5.0.95.orig/mysql-test/std_data/cacert.pem mysql-5.0.95/mysql-test/std_data/cacert.pem
--- mysql-5.0.95.orig/mysql-test/std_data/cacert.pem	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/std_data/cacert.pem	2012-02-03 13:47:15.786970891 -0500
@@ -1,17 +1,22 @@
 -----BEGIN CERTIFICATE-----
-MIICrTCCAhagAwIBAgIJAMI7xZKjhrDbMA0GCSqGSIb3DQEBBAUAMEQxCzAJBgNV
+MIIDsjCCApqgAwIBAgIJAL5YrUwfPSWVMA0GCSqGSIb3DQEBBQUAMEQxCzAJBgNV
 BAYTAlNFMRAwDgYDVQQIEwdVcHBzYWxhMRAwDgYDVQQHEwdVcHBzYWxhMREwDwYD
-VQQKEwhNeVNRTCBBQjAeFw0xMDAxMjkxMTQ3MTBaFw0xNTAxMjgxMTQ3MTBaMEQx
+VQQKEwhNeVNRTCBBQjAeFw0xMDAxMjkwNTU5NTNaFw0xNTAxMjgwNTU5NTNaMEQx
 CzAJBgNVBAYTAlNFMRAwDgYDVQQIEwdVcHBzYWxhMRAwDgYDVQQHEwdVcHBzYWxh
-MREwDwYDVQQKEwhNeVNRTCBBQjCBnzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEA
-wQYsOEfrN4ESP3FjsI8cghE+tZVuyK2gck61lwieVxjgFMtBd65mI5a1y9pmlOI1
-yM4SB2Ppqcuw7/e1CdV1y7lvHrGNt5yqEHbN4QX1gvsN8TQauP/2WILturk4R4Hq
-rKg0ZySu7f1Xhl0ed9a48LpaEHD17IcxWEGMMJwAxF0CAwEAAaOBpjCBozAMBgNV
-HRMEBTADAQH/MB0GA1UdDgQWBBSvktYQ0ahLnyxyVKqty+WpBbBrDTB0BgNVHSME
-bTBrgBSvktYQ0ahLnyxyVKqty+WpBbBrDaFIpEYwRDELMAkGA1UEBhMCU0UxEDAO
-BgNVBAgTB1VwcHNhbGExEDAOBgNVBAcTB1VwcHNhbGExETAPBgNVBAoTCE15U1FM
-IEFCggkAwjvFkqOGsNswDQYJKoZIhvcNAQEEBQADgYEAdKN1PjwMHAKG2Ww1145g
-JQGBnKxSFOUaoSvkBi/4ntTM+ysnViWh7WvxyWjR9zU9arfr7aqsDeQxm0XDOqzj
-AQ/cQIla2/Li8tXyfc06bisH/IHRaSc2zWqioTKbEwMdVOdrvq4a8V8ic3xYyIWn
-7F4WeS07J8LKardSvM0+hOA=
+MREwDwYDVQQKEwhNeVNRTCBBQjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoC
+ggEBAL6kNN4peX7uhK9rb06W/QbPEpVuejmdWdl2PqMshP/eSuXXw7kwVgfpxx9R
+vC000CKQQSG9MCoZjtqPnFRsetmWLZgApRpEalGXTXJqq9sEbCfoFizg94U8G7d2
+u5XJjLVmcG34ru36KoBgVx1zeH1puBAf8dOzrE4L7Y+ZQBFzFohjh8C2LqWC4nM5
+qsLmOkDWMipGqYU5DvkKjIbTbwTyRNRgZHWSPfVDDPUIUOsY4BGUp2DpgeGY9aEv
+lIs57Ev9JqlIUCV65lOhhDkG+xwmkHKHA+ECEU9cALI8+uXbh48MB9XpMOuk408X
+/lX89aZwD0/G9kmObVGnE2G+H5UCAwEAAaOBpjCBozAdBgNVHQ4EFgQUsft+d7VA
+jWgRftkR5cPG2k2sUbAwdAYDVR0jBG0wa4AUsft+d7VAjWgRftkR5cPG2k2sUbCh
+SKRGMEQxCzAJBgNVBAYTAlNFMRAwDgYDVQQIEwdVcHBzYWxhMRAwDgYDVQQHEwdV
+cHBzYWxhMREwDwYDVQQKEwhNeVNRTCBBQoIJAL5YrUwfPSWVMAwGA1UdEwQFMAMB
+Af8wDQYJKoZIhvcNAQEFBQADggEBALRUOAmdL8R8sl1y8kiEiFgDatdXK5RDqWai
+8yZChfmwTIToHhmQsOEshJe2e8hky3huUj+33VyXjINoMbebIwMuXPwEkbJal8RZ
+nSJmF0jN1Qz7J/jFffwK9xmejWZJx49Kt2+Qwrwp6kDeq9TLFqQOoVczgyJPYsTL
+NAOib5WqTud3XWvCwxrhqmWu7JZq6sp1fomP/uunprb8y2miWfLESZN2mKAhm44Q
+Lws867LT8v2lskEjq2dT1LutD5+R66XcdjgSr0uDziDs64jZwCD6ea94hVFM7ej0
+ZOXYeSEZJ56FjUxu632e9fY8NyMh30yKjjmQf1mM9PuGJvdvsWU=
 -----END CERTIFICATE-----
diff -Naur mysql-5.0.95.orig/mysql-test/std_data/client-cert.pem mysql-5.0.95/mysql-test/std_data/client-cert.pem
--- mysql-5.0.95.orig/mysql-test/std_data/client-cert.pem	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/std_data/client-cert.pem	2012-02-03 13:47:19.307001834 -0500
@@ -1,46 +1,70 @@
 Certificate:
     Data:
-        Version: 1 (0x0)
-        Serial Number: 1048577 (0x100001)
-        Signature Algorithm: md5WithRSAEncryption
+        Version: 3 (0x2)
+        Serial Number: 3 (0x3)
+        Signature Algorithm: sha1WithRSAEncryption
         Issuer: C=SE, ST=Uppsala, L=Uppsala, O=MySQL AB
         Validity
-            Not Before: Jan 29 11:50:22 2010 GMT
-            Not After : Jan 28 11:50:22 2015 GMT
-        Subject: C=SE, ST=Uppsala, O=MySQL AB
+            Not Before: Jan 29 06:07:48 2010 GMT
+            Not After : Aug 12 06:07:48 2030 GMT
+        Subject: C=SE, ST=Uppsala, O=MySQL AB/emailAddress=abstract.mysql.developer@mysql.com
         Subject Public Key Info:
             Public Key Algorithm: rsaEncryption
-                Public-Key: (1024 bit)
-                Modulus:
-                    00:cc:9a:37:49:13:66:dc:cf:e3:0b:13:a1:23:ed:
-                    78:db:4e:bd:11:f6:8c:0d:76:f9:a3:32:56:9a:f8:
-                    a1:21:6a:55:4e:4d:3f:e6:67:9d:26:99:b2:cd:a4:
-                    9a:d2:2b:59:5c:d7:8a:d3:60:68:f8:18:bd:c5:be:
-                    15:e1:2a:3c:a3:d4:61:cb:f5:11:94:17:81:81:f7:
-                    87:8c:f6:6a:d2:ee:d8:e6:77:f6:62:66:4d:2e:16:
-                    8d:08:81:4a:c9:c6:4b:31:e5:b9:c7:8a:84:96:48:
-                    a7:47:8c:0d:26:90:56:4e:e6:a5:6e:8c:b3:f2:9f:
-                    fc:3d:78:9b:49:6e:86:83:77
+            RSA Public Key: (1024 bit)
+                Modulus (1024 bit):
+                    00:b0:27:bc:06:b1:54:c0:85:7d:dc:fd:2f:ee:46:
+                    57:b2:66:4d:4a:6d:a4:54:6d:9d:9a:e3:36:b4:39:
+                    3b:12:2d:0b:79:45:cb:49:a3:ab:32:6a:65:fb:86:
+                    3e:bc:d2:84:a6:66:1a:0b:68:60:1f:e4:f9:ec:2f:
+                    79:8a:a1:97:a9:e5:64:66:16:4b:5a:89:f2:c5:c4:
+                    23:e5:28:92:4a:31:6c:03:33:9a:99:13:db:4d:c2:
+                    04:4d:cb:32:1e:59:d0:95:bd:3a:55:e6:0e:83:40:
+                    62:59:03:c4:42:38:6c:14:85:b2:5f:f3:6b:a4:bd:
+                    ab:f0:78:04:1d:f0:59:d8:b7
                 Exponent: 65537 (0x10001)
-    Signature Algorithm: md5WithRSAEncryption
-        5e:1f:a3:53:5f:24:13:1c:f8:28:32:b0:7f:69:69:f3:0e:c0:
-        34:87:10:03:7d:da:15:8b:bd:19:b8:1a:56:31:e7:85:49:81:
-        c9:7f:45:20:74:3e:89:c0:e0:26:84:51:cc:04:16:ce:69:99:
-        01:e1:26:99:b3:e3:f5:bd:ec:5f:a0:84:e4:38:da:75:78:7b:
-        89:9c:d2:cd:60:95:20:ba:8e:e3:7c:e6:df:76:3a:7c:89:77:
-        02:94:86:11:3a:c4:61:7d:6f:71:83:21:8a:17:fb:17:e2:ee:
-        02:6b:61:c1:b4:52:63:d7:d8:46:b2:c5:9c:6f:38:91:8a:35:
-        32:0b
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:FALSE
+            X509v3 Subject Key Identifier: 
+                1B:80:A5:E8:DC:7F:8E:F3:1F:C8:4A:43:DA:C0:99:A1:4B:E7:63:FA
+            X509v3 Authority Key Identifier: 
+                keyid:B1:FB:7E:77:B5:40:8D:68:11:7E:D9:11:E5:C3:C6:DA:4D:AC:51:B0
+                DirName:/C=SE/ST=Uppsala/L=Uppsala/O=MySQL AB
+                serial:BE:58:AD:4C:1F:3D:25:95
+
+    Signature Algorithm: sha1WithRSAEncryption
+        45:90:3d:ee:23:6d:64:7d:2f:05:42:9b:f0:18:ad:af:64:75:
+        6a:14:e4:39:ff:2b:9f:76:8c:16:52:d5:04:85:72:d8:9f:e7:
+        3d:f0:a2:31:44:c6:4a:59:34:81:e9:85:a7:e3:73:df:6f:dc:
+        37:71:57:ba:7c:b0:56:2a:07:88:6c:f7:92:d3:dc:43:41:0f:
+        ed:e3:3c:34:ed:a5:0f:25:cd:ba:76:a4:b5:aa:76:ec:a9:1b:
+        64:f4:75:d4:20:21:27:78:b1:b2:bd:10:20:9e:0c:c2:ea:40:
+        09:6a:f9:31:7b:d2:47:32:58:54:0e:e7:a1:49:a0:24:ca:26:
+        56:3b:14:9b:a2:66:6a:28:68:49:25:bd:68:8e:31:3b:cb:04:
+        7c:0b:d4:62:f8:95:a3:78:0d:4c:77:be:81:24:09:f0:e7:f0:
+        3e:d2:92:6f:4b:fc:34:1a:76:fa:ae:03:18:c0:11:b6:a6:63:
+        29:cd:09:e3:11:69:3a:de:7d:74:2d:ed:60:d6:0b:ba:09:a7:
+        17:39:15:84:c1:8b:9b:cd:05:8f:be:58:ba:d1:f4:62:9f:e7:
+        18:8f:4d:68:dd:d1:ca:04:80:8d:03:e6:36:93:68:58:ae:26:
+        30:6b:e6:36:8a:3a:1d:1b:29:3f:c3:d2:18:0d:af:ef:6a:0f:
+        f5:0e:9a:3b
 -----BEGIN CERTIFICATE-----
-MIIB5zCCAVACAxAAATANBgkqhkiG9w0BAQQFADBEMQswCQYDVQQGEwJTRTEQMA4G
-A1UECBMHVXBwc2FsYTEQMA4GA1UEBxMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwg
-QUIwHhcNMTAwMTI5MTE1MDIyWhcNMTUwMTI4MTE1MDIyWjAyMQswCQYDVQQGEwJT
-RTEQMA4GA1UECBMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwgQUIwgZ8wDQYJKoZI
-hvcNAQEBBQADgY0AMIGJAoGBAMyaN0kTZtzP4wsToSPteNtOvRH2jA12+aMyVpr4
-oSFqVU5NP+ZnnSaZss2kmtIrWVzXitNgaPgYvcW+FeEqPKPUYcv1EZQXgYH3h4z2
-atLu2OZ39mJmTS4WjQiBSsnGSzHluceKhJZIp0eMDSaQVk7mpW6Ms/Kf/D14m0lu
-hoN3AgMBAAEwDQYJKoZIhvcNAQEEBQADgYEAXh+jU18kExz4KDKwf2lp8w7ANIcQ
-A33aFYu9GbgaVjHnhUmByX9FIHQ+icDgJoRRzAQWzmmZAeEmmbPj9b3sX6CE5Dja
-dXh7iZzSzWCVILqO43zm33Y6fIl3ApSGETrEYX1vcYMhihf7F+LuAmthwbRSY9fY
-RrLFnG84kYo1Mgs=
+MIIDRDCCAiygAwIBAgIBAzANBgkqhkiG9w0BAQUFADBEMQswCQYDVQQGEwJTRTEQ
+MA4GA1UECBMHVXBwc2FsYTEQMA4GA1UEBxMHVXBwc2FsYTERMA8GA1UEChMITXlT
+UUwgQUIwHhcNMTAwMTI5MDYwNzQ4WhcNMzAwODEyMDYwNzQ4WjBlMQswCQYDVQQG
+EwJTRTEQMA4GA1UECBMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwgQUIxMTAvBgkq
+hkiG9w0BCQEWImFic3RyYWN0Lm15c3FsLmRldmVsb3BlckBteXNxbC5jb20wgZ8w
+DQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBALAnvAaxVMCFfdz9L+5GV7JmTUptpFRt
+nZrjNrQ5OxItC3lFy0mjqzJqZfuGPrzShKZmGgtoYB/k+ewveYqhl6nlZGYWS1qJ
+8sXEI+UokkoxbAMzmpkT203CBE3LMh5Z0JW9OlXmDoNAYlkDxEI4bBSFsl/za6S9
+q/B4BB3wWdi3AgMBAAGjgaMwgaAwCQYDVR0TBAIwADAdBgNVHQ4EFgQUG4Cl6Nx/
+jvMfyEpD2sCZoUvnY/owdAYDVR0jBG0wa4AUsft+d7VAjWgRftkR5cPG2k2sUbCh
+SKRGMEQxCzAJBgNVBAYTAlNFMRAwDgYDVQQIEwdVcHBzYWxhMRAwDgYDVQQHEwdV
+cHBzYWxhMREwDwYDVQQKEwhNeVNRTCBBQoIJAL5YrUwfPSWVMA0GCSqGSIb3DQEB
+BQUAA4IBAQBFkD3uI21kfS8FQpvwGK2vZHVqFOQ5/yufdowWUtUEhXLYn+c98KIx
+RMZKWTSB6YWn43Pfb9w3cVe6fLBWKgeIbPeS09xDQQ/t4zw07aUPJc26dqS1qnbs
+qRtk9HXUICEneLGyvRAgngzC6kAJavkxe9JHMlhUDuehSaAkyiZWOxSbomZqKGhJ
+Jb1ojjE7ywR8C9Ri+JWjeA1Md76BJAnw5/A+0pJvS/w0Gnb6rgMYwBG2pmMpzQnj
+EWk63n10Le1g1gu6CacXORWEwYubzQWPvli60fRin+cYj01o3dHKBICNA+Y2k2hY
+riYwa+Y2ijodGyk/w9IYDa/vag/1Dpo7
 -----END CERTIFICATE-----
diff -Naur mysql-5.0.95.orig/mysql-test/std_data/client-key.pem mysql-5.0.95/mysql-test/std_data/client-key.pem
--- mysql-5.0.95.orig/mysql-test/std_data/client-key.pem	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/std_data/client-key.pem	2012-02-03 13:47:21.595021940 -0500
@@ -1,15 +1,15 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIICXQIBAAKBgQDMmjdJE2bcz+MLE6Ej7XjbTr0R9owNdvmjMlaa+KEhalVOTT/m
-Z50mmbLNpJrSK1lc14rTYGj4GL3FvhXhKjyj1GHL9RGUF4GB94eM9mrS7tjmd/Zi
-Zk0uFo0IgUrJxksx5bnHioSWSKdHjA0mkFZO5qVujLPyn/w9eJtJboaDdwIDAQAB
-AoGASqk/4We2En+93y3jkIO4pXafIe3w/3zZ7caRue1ehx4RUQh5d+95djuB9u7J
-HEZ7TpjM7QNyao5EueL6gvbxt0LXFvqAMni7yM9tt/HUYtHHPqYiRtUny9bKYFTm
-l8szCCMal/wD9GZU9ByHDNHm7tHUMyMhARNTYSgx+SERFmECQQD/6jJocC4SXf6f
-T3LqimWR02lbJ7qCoDgRglsUXh0zjrG+IIiAyE+QOCCx1GMe3Uw6bsIuYwdHT6as
-WcdPs04xAkEAzKulvEvLVvN5zfa/DTYRTV7jh6aDleOxjsD5oN/oJXoACnPzVuUL
-qQQMNtuAXm6Q1QItrRxpQsSKbY0UQka6JwJBAOSgoNoG5lIIYTKIMvzwGV+XBLeo
-HYsXgh+6Wo4uql3mLErUG78ZtWL9kc/tE4R+ZdyKGLaCR/1gXmH5bwN4B/ECQEBb
-uUH8k3REG4kojesZlVc+/00ojzgS4UKCa/yqa9VdB6ZBz8MDQydinnShkTwgiGpy
-xOoqhO753o2UT0qH8wECQQC99IEJWUnwvExVMkLaZH5NjAFJkb22sjkmuT11tAgU
-RQgOMoDOm6driojnOnDWOkx1r1Gy9NgMLooduja4v6cx
+MIICXAIBAAKBgQCwJ7wGsVTAhX3c/S/uRleyZk1KbaRUbZ2a4za0OTsSLQt5RctJ
+o6syamX7hj680oSmZhoLaGAf5PnsL3mKoZep5WRmFktaifLFxCPlKJJKMWwDM5qZ
+E9tNwgRNyzIeWdCVvTpV5g6DQGJZA8RCOGwUhbJf82ukvavweAQd8FnYtwIDAQAB
+AoGAdDGxgASmr9sd+4/ydsxlemKqpuaNl7c6V3l6kwSRJ1sZD2RvRGK2QSoXnQKf
+DXQbVZtRoKa3iTJDfHyQQlZ2r9z2T2zEuxu6x/lUJO4EhcLPGK53Ew5BP+t/EqQe
+xfq+dwZ/NLl7SkBFf2FKPtb0juiT7cTfwjb0VHtaBGlNcGkCQQDgGmqAnpEVoaca
+9juAFz6C4KD4BsaKuMUsJWEPohRACAzBI0tyGpLyPaIep7a5F1KTMsfDaKIFr8oh
+Bmx7L5jbAkEAyTpAc1gvyfLtToG2IPjOND7EvzlNY/kMmt/43BPfknbRyJLWgGOO
+pJ4CiAhSGEBSXvjEcwoIlUdzYV6Eu7PIVQJAKKMhpOkcDfwePTlhWLzeTQVGqzMv
+VYBj4Ll2fEs7uNC831jzGWHQO7UJbN2kuLON+bs/liOnHbjewlZT3ZKn+QJBAJth
+NJUcOkJ5J06gkC7jj+39rdRufdi9XUNVwIbHfbJNt21mHOWkemeYbyH1MtbxC+mz
+s5WU4M3MD0bXQiziBMUCQAjb86wy6/ktxgi/GQHV0haVJSK8pqmDRz6m/1m9efEK
+iXeeaxYiWC07vu+NLy/ysq48jOMURD/m3GbFDGIcnRU=
 -----END RSA PRIVATE KEY-----
diff -Naur mysql-5.0.95.orig/mysql-test/std_data/server-cert.pem mysql-5.0.95/mysql-test/std_data/server-cert.pem
--- mysql-5.0.95.orig/mysql-test/std_data/server-cert.pem	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/std_data/server-cert.pem	2012-02-03 13:45:16.687922618 -0500
@@ -1,41 +1,70 @@
 Certificate:
     Data:
-        Version: 1 (0x0)
-        Serial Number: 1048578 (0x100002)
-        Signature Algorithm: md5WithRSAEncryption
+        Version: 3 (0x2)
+        Serial Number: 1 (0x1)
+        Signature Algorithm: sha1WithRSAEncryption
         Issuer: C=SE, ST=Uppsala, L=Uppsala, O=MySQL AB
         Validity
-            Not Before: Jan 29 11:56:49 2010 GMT
-            Not After : Jan 28 11:56:49 2015 GMT
-        Subject: C=SE, ST=Uppsala, O=MySQL AB, CN=localhost
+            Not Before: Jan 29 06:02:25 2010 GMT
+            Not After : Aug 12 06:02:25 2030 GMT
+        Subject: C=SE, ST=Uppsala, O=MySQL AB, CN=localhost/emailAddress=abstract.mysql.developer@mysql.com
         Subject Public Key Info:
             Public Key Algorithm: rsaEncryption
-                Public-Key: (512 bit)
-                Modulus:
-                    00:cd:e4:87:51:9d:72:11:a0:d1:fa:f3:92:8b:13:
-                    1c:eb:f7:e2:9a:2f:72:a8:d6:65:48:d1:69:af:1b:
-                    c0:4c:13:e5:60:60:51:41:e9:ab:a6:bc:13:bb:0c:
-                    5e:32:7c:d9:6c:9e:cd:05:24:84:78:db:80:91:2e:
-                    d8:88:2b:c2:ed
+            RSA Public Key: (1024 bit)
+                Modulus (1024 bit):
+                    00:a1:4e:1b:2d:7a:71:9a:81:70:cf:9b:0e:32:aa:
+                    83:c7:2d:bb:30:a4:cc:16:f6:68:57:49:37:3f:ca:
+                    b5:47:83:14:ca:4b:04:fe:a9:18:fd:ba:6f:2f:15:
+                    2b:d7:da:b1:cd:e5:43:93:64:4d:b1:4e:d1:5a:10:
+                    e3:5e:81:dd:2a:72:ff:74:a2:eb:9a:3b:9a:b2:46:
+                    ac:01:55:7e:80:b1:92:0d:e7:c1:29:ff:0e:90:ae:
+                    5c:cc:66:af:f8:c7:74:1d:4f:3f:e5:2c:0b:cb:f1:
+                    78:06:ee:da:6f:63:b6:80:dc:30:fe:ef:d5:7d:fc:
+                    92:f8:74:5f:33:33:a7:65:37
                 Exponent: 65537 (0x10001)
-    Signature Algorithm: md5WithRSAEncryption
-        73:ce:9c:6e:39:46:b4:14:be:da:3f:f3:1b:ba:90:bc:23:43:
-        d7:82:2a:70:4e:a6:d9:5a:65:5c:b7:df:71:df:75:77:c5:80:
-        a4:af:fa:d2:59:e2:fd:c9:9c:f0:98:95:8e:69:a9:8c:7c:d8:
-        6f:48:d2:e3:36:e0:cd:ff:3f:d1:a5:e6:ab:75:09:c4:50:10:
-        c4:96:dd:bf:3b:de:32:46:da:ca:4a:f1:d6:52:8a:33:2f:ab:
-        f5:2e:70:3f:d4:9c:be:00:c8:03:f9:39:8a:df:5b:70:3c:40:
-        ef:03:be:7c:3d:1d:32:32:f3:51:81:e2:83:30:6e:3d:38:9b:
-        fb:3c
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:FALSE
+            X509v3 Subject Key Identifier: 
+                AA:6D:65:A9:98:45:49:56:A1:41:96:10:9A:BF:05:0B:A7:86:AC:10
+            X509v3 Authority Key Identifier: 
+                keyid:B1:FB:7E:77:B5:40:8D:68:11:7E:D9:11:E5:C3:C6:DA:4D:AC:51:B0
+                DirName:/C=SE/ST=Uppsala/L=Uppsala/O=MySQL AB
+                serial:BE:58:AD:4C:1F:3D:25:95
+
+    Signature Algorithm: sha1WithRSAEncryption
+        5e:ea:d1:ae:3f:d6:ab:fc:e9:1c:e4:70:8f:c9:73:86:a8:c4:
+        1e:6f:f3:cb:c0:c7:9f:55:81:4e:8d:ce:0a:7b:fb:d6:be:15:
+        cc:4f:18:f7:51:d6:e8:51:ec:97:22:e2:f7:bd:b2:e9:8f:da:
+        59:ec:b2:39:02:73:b7:fd:71:90:ac:cd:49:9e:53:ab:66:ca:
+        28:dc:71:c5:6c:d5:be:c1:39:15:04:93:ed:d8:fb:84:56:4d:
+        46:f1:75:5a:9b:0c:ee:b1:c9:3e:4d:15:3b:5e:ba:68:3a:62:
+        c0:3c:e6:63:b6:6d:53:3c:82:d6:62:7c:0e:c3:e4:d0:73:69:
+        34:a9:ce:7e:af:fc:1a:30:e1:ad:00:c1:52:c4:2a:88:04:da:
+        fc:92:3b:a6:0b:7c:e6:14:5a:b6:f4:9b:74:e1:23:a5:c7:2c:
+        18:83:98:18:32:4c:c0:9b:1b:c4:94:14:92:24:50:f5:d2:88:
+        db:a2:8d:80:bb:df:2b:44:5c:27:18:3a:d6:b2:2f:2e:71:47:
+        cb:f0:46:59:e6:cf:40:23:f4:72:72:36:00:0d:86:a4:f0:a1:
+        86:86:0d:07:9f:4c:f9:c4:c4:83:ed:d9:33:e2:ea:4d:57:28:
+        cc:2b:fb:5c:74:38:e0:ed:24:6d:32:1b:93:e2:bf:9f:a2:dd:
+        39:b2:ce:54
 -----BEGIN CERTIFICATE-----
-MIIBtzCCASACAxAAAjANBgkqhkiG9w0BAQQFADBEMQswCQYDVQQGEwJTRTEQMA4G
-A1UECBMHVXBwc2FsYTEQMA4GA1UEBxMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwg
-QUIwHhcNMTAwMTI5MTE1NjQ5WhcNMTUwMTI4MTE1NjQ5WjBGMQswCQYDVQQGEwJT
-RTEQMA4GA1UECBMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwgQUIxEjAQBgNVBAMT
-CWxvY2FsaG9zdDBcMA0GCSqGSIb3DQEBAQUAA0sAMEgCQQDN5IdRnXIRoNH685KL
-Exzr9+KaL3Ko1mVI0WmvG8BME+VgYFFB6aumvBO7DF4yfNlsns0FJIR424CRLtiI
-K8LtAgMBAAEwDQYJKoZIhvcNAQEEBQADgYEAc86cbjlGtBS+2j/zG7qQvCND14Iq
-cE6m2VplXLffcd91d8WApK/60lni/cmc8JiVjmmpjHzYb0jS4zbgzf8/0aXmq3UJ
-xFAQxJbdvzveMkbaykrx1lKKMy+r9S5wP9ScvgDIA/k5it9bcDxA7wO+fD0dMjLz
-UYHigzBuPTib+zw=
+MIIDWDCCAkCgAwIBAgIBATANBgkqhkiG9w0BAQUFADBEMQswCQYDVQQGEwJTRTEQ
+MA4GA1UECBMHVXBwc2FsYTEQMA4GA1UEBxMHVXBwc2FsYTERMA8GA1UEChMITXlT
+UUwgQUIwHhcNMTAwMTI5MDYwMjI1WhcNMzAwODEyMDYwMjI1WjB5MQswCQYDVQQG
+EwJTRTEQMA4GA1UECBMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwgQUIxEjAQBgNV
+BAMTCWxvY2FsaG9zdDExMC8GCSqGSIb3DQEJARYiYWJzdHJhY3QubXlzcWwuZGV2
+ZWxvcGVyQG15c3FsLmNvbTCBnzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEAoU4b
+LXpxmoFwz5sOMqqDxy27MKTMFvZoV0k3P8q1R4MUyksE/qkY/bpvLxUr19qxzeVD
+k2RNsU7RWhDjXoHdKnL/dKLrmjuaskasAVV+gLGSDefBKf8OkK5czGav+Md0HU8/
+5SwLy/F4Bu7ab2O2gNww/u/VffyS+HRfMzOnZTcCAwEAAaOBozCBoDAJBgNVHRME
+AjAAMB0GA1UdDgQWBBSqbWWpmEVJVqFBlhCavwULp4asEDB0BgNVHSMEbTBrgBSx
++353tUCNaBF+2RHlw8baTaxRsKFIpEYwRDELMAkGA1UEBhMCU0UxEDAOBgNVBAgT
+B1VwcHNhbGExEDAOBgNVBAcTB1VwcHNhbGExETAPBgNVBAoTCE15U1FMIEFCggkA
+vlitTB89JZUwDQYJKoZIhvcNAQEFBQADggEBAF7q0a4/1qv86RzkcI/Jc4aoxB5v
+88vAx59VgU6Nzgp7+9a+FcxPGPdR1uhR7Jci4ve9sumP2lnssjkCc7f9cZCszUme
+U6tmyijcccVs1b7BORUEk+3Y+4RWTUbxdVqbDO6xyT5NFTteumg6YsA85mO2bVM8
+gtZifA7D5NBzaTSpzn6v/Bow4a0AwVLEKogE2vySO6YLfOYUWrb0m3ThI6XHLBiD
+mBgyTMCbG8SUFJIkUPXSiNuijYC73ytEXCcYOtayLy5xR8vwRlnmz0Aj9HJyNgAN
+hqTwoYaGDQefTPnExIPt2TPi6k1XKMwr+1x0OODtJG0yG5Piv5+i3TmyzlQ=
 -----END CERTIFICATE-----
diff -Naur mysql-5.0.95.orig/mysql-test/std_data/server-key.pem mysql-5.0.95/mysql-test/std_data/server-key.pem
--- mysql-5.0.95.orig/mysql-test/std_data/server-key.pem	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/std_data/server-key.pem	2012-02-03 13:45:17.463931567 -0500
@@ -1,9 +1,15 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIIBOwIBAAJBAM3kh1GdchGg0frzkosTHOv34povcqjWZUjRaa8bwEwT5WBgUUHp
-q6a8E7sMXjJ82WyezQUkhHjbgJEu2Igrwu0CAwEAAQJBAJuwhFbF3NzRpBbEmnqJ
-4GPa1UJMQMLFJF+04tqj/HxJcAIVhOJhGmmtYNw1yjz/ZsPnfJCMz4eFOtdjvGtf
-peECIQDmFFg2WLvYo+2m9w9V7z4ZIkg7ixYkI/ObUUctfZkPOQIhAOUWnrvjFrAX
-bIvYT/YR50+3ZDLEc51XxNgJnWqWYl1VAiEAnTOFWgyivFC1DgF8PvDp8u5TgCt2
-A1d1GMgd490O+TECIC/WMl0/hTxOF9930vKqOGf//o9PUGkZq8QE9fcM4gtlAiAE
-iOcFpnLjtWj57jrhuw214ucnB5rklkQQe+AtcARNkg==
+MIICWwIBAAKBgQChThstenGagXDPmw4yqoPHLbswpMwW9mhXSTc/yrVHgxTKSwT+
+qRj9um8vFSvX2rHN5UOTZE2xTtFaEONegd0qcv90ouuaO5qyRqwBVX6AsZIN58Ep
+/w6QrlzMZq/4x3QdTz/lLAvL8XgG7tpvY7aA3DD+79V9/JL4dF8zM6dlNwIDAQAB
+AoGAcxeQyrVH448iF+8xhTMnDXHy0zVyrZupPUZvVmPjTqAXsHMwoCe1PizM8hIw
+EzbmkNtZiQwTZ0SumY3dPWx5iZM0P+ufoxM1Asendgr9EUg0udiEmBQIyeXvzwR/
+gx5OyZYXR7r+H4pr+3Aqd4Lia1CBNogUZMFq1OR8tZquvJECQQDPEcAZyWbRZLr3
+LLhCKmuzt+FKU+2k/xk0UfLegCkmZ5+qrc0s9fbev2VpNvJDUg5y8gbP+zK7Y5cy
+Eh6VBKXvAkEAx2vwX9A5oSSYVpMZkGVrMaRyIA4LdxlC31HKXCxUzm28z1x7QuDu
+8BdZPUwSRTtNyUCB5daZd2tQjdkSf7O9OQJAHSjlwXvgM8qYnag5hm/M3PEfE8V/
+6MVXukt0BStNIDBlhraU9yjoC/QuBsQLF7ndeQhmNzGrskHTD+UHU3rlFwJAay7A
+iz7HOq0Ik7+MRee8rXnNPR1PXDvfjVfQg6nwCuNAT9NNDK0Iv1JNTHzOdRdiv4BE
+UNv+sAilyxeYsbpRQQJAMMf88+cU7Gb0cMKMlS5qM5UyIbT2lu8OypXC7U/iN74V
+EAh4dIpLYUFa8dOkO/eP5lntIb/GlW63rMBNj0vQBQ==
 -----END RSA PRIVATE KEY-----
diff -Naur mysql-5.0.95.orig/mysql-test/std_data/server8k-cert.pem mysql-5.0.95/mysql-test/std_data/server8k-cert.pem
--- mysql-5.0.95.orig/mysql-test/std_data/server8k-cert.pem	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/std_data/server8k-cert.pem	2012-02-03 13:45:18.150937177 -0500
@@ -1,125 +1,69 @@
 Certificate:
     Data:
-        Version: 1 (0x0)
-        Serial Number: 1048579 (0x100003)
-        Signature Algorithm: md5WithRSAEncryption
+        Version: 3 (0x2)
+        Serial Number: 2 (0x2)
+        Signature Algorithm: sha1WithRSAEncryption
         Issuer: C=SE, ST=Uppsala, L=Uppsala, O=MySQL AB
         Validity
-            Not Before: Jan 29 12:01:53 2010 GMT
-            Not After : Jan 28 12:01:53 2015 GMT
+            Not Before: Jan 29 06:04:32 2010 GMT
+            Not After : Aug 12 06:04:32 2030 GMT
         Subject: C=SE, ST=Uppsala, O=MySQL AB, CN=server
         Subject Public Key Info:
             Public Key Algorithm: rsaEncryption
-                Public-Key: (8192 bit)
-                Modulus:
-                    00:ca:aa:1d:c4:11:ec:91:f0:c7:ff:5f:90:92:fc:
-                    40:0c:5e:b7:3d:00:c5:20:d5:0f:89:31:07:d7:41:
-                    4c:8b:60:80:aa:38:14:de:93:6b:9c:74:88:41:68:
-                    b5:02:41:01:2d:86:a2:7a:95:53:5e:7b:67:2f:6c:
-                    1e:29:51:f9:44:fd:4a:80:be:b2:23:a1:3e:1b:38:
-                    cf:88:c4:71:ee:f8:6b:41:c5:2d:c0:c3:52:ac:59:
-                    7d:81:34:19:95:32:b8:9a:51:b6:41:36:d4:c4:a1:
-                    ae:84:e6:38:b9:e8:bf:96:be:19:7a:6b:77:4d:e0:
-                    de:e6:b3:b6:6b:bc:3d:dd:68:bc:4b:c4:eb:f5:36:
-                    93:ed:56:a2:15:50:8a:10:e8:d6:22:ed:6c:b1:cd:
-                    c3:18:c9:f6:0a:e1:de:61:65:62:d6:14:41:8c:b5:
-                    fb:14:68:c1:cf:12:5d:41:21:9d:57:11:43:7d:bb:
-                    43:2c:21:bb:c3:44:7d:a8:cf:1f:c3:71:75:b5:47:
-                    c2:7d:ce:38:3c:73:64:9e:15:d8:a7:27:cf:bd:40:
-                    c8:45:08:e3:c8:39:a8:0b:8e:c2:5b:7b:f1:47:91:
-                    12:91:cc:e1:00:e0:94:5b:bd:32:e4:0c:8d:c3:be:
-                    cc:76:32:52:12:69:b0:18:e0:b0:c2:76:34:5a:5f:
-                    79:d9:f6:81:9d:02:0a:61:69:1c:33:ce:49:fa:76:
-                    03:1e:07:5b:27:0b:bf:34:9e:34:96:b8:03:9b:50:
-                    3a:6a:2f:17:7a:14:cf:65:63:00:37:52:a8:73:ce:
-                    4b:14:40:f4:d2:9a:56:54:33:b8:77:2e:42:5b:8f:
-                    ec:1f:18:f4:ad:ab:8a:4a:8d:6d:70:25:f3:58:e7:
-                    cb:66:51:14:7d:16:f4:eb:6d:56:76:76:51:6e:d6:
-                    1d:da:d3:8d:c0:64:5a:67:4e:af:e2:bf:33:d1:b8:
-                    f6:2a:fc:57:87:a7:35:5e:80:c9:ac:fc:87:c9:71:
-                    17:91:bf:b7:4d:a3:ed:3c:1b:27:f4:66:a0:f9:46:
-                    03:27:cc:ea:80:f6:4b:40:f6:41:94:cd:bd:0a:b3:
-                    ef:26:be:de:6f:69:ae:0f:3f:1c:55:63:33:90:9b:
-                    ed:ca:5a:12:4d:de:4b:06:c2:a2:92:b0:42:3d:31:
-                    af:a4:15:12:15:f8:8a:e9:88:8d:cf:fd:85:66:50:
-                    6f:11:f1:9f:48:f3:b5:ba:9d:86:68:24:a2:5d:a8:
-                    7c:54:42:fa:d8:b5:c5:f2:dd:0e:0f:d0:68:e4:54:
-                    7e:c5:b9:a0:9b:65:2d:77:f4:8f:b9:30:0a:d5:86:
-                    5c:ed:c9:7c:d1:da:9d:0d:63:50:ee:e5:1e:92:63:
-                    cc:a2:0c:e8:4a:96:02:4d:dc:8f:df:7c:8f:08:18:
-                    a8:30:88:d7:af:89:ad:fc:57:4b:10:f9:f1:cb:48:
-                    e8:b6:3b:c8:3f:fc:c2:d3:d1:4a:10:3c:1b:6b:64:
-                    dc:e5:65:1e:5b:b2:da:b1:e2:24:97:8f:ee:c0:4b:
-                    8e:18:83:7c:17:a6:3c:45:b3:60:06:23:f2:2f:18:
-                    13:9e:17:8a:c6:72:79:8c:4d:04:f3:9d:ea:e0:25:
-                    d3:33:8c:1e:11:47:63:1f:a5:45:3f:bd:85:b3:fe:
-                    a5:68:ee:48:b7:0c:a4:c9:7f:72:d0:75:66:9b:6a:
-                    f9:a0:50:f3:a8:59:6d:a3:dd:38:4f:70:2b:bb:ff:
-                    92:2e:71:ab:ef:e9:00:ed:0d:d1:b4:6f:f0:8e:b2:
-                    09:fb:4d:61:0d:d9:10:d5:54:11:cd:03:94:84:fd:
-                    a8:68:e4:45:6e:1e:6a:1e:2f:85:a1:6d:f5:b6:c0:
-                    f1:ee:f7:36:e9:fe:c2:f7:ad:cc:13:46:5b:88:42:
-                    f0:2d:1f:b5:0e:7e:b5:2b:e4:8d:ab:b9:87:30:6a:
-                    3d:12:f4:ad:f3:1c:ac:cc:1a:48:29:2a:96:7b:80:
-                    00:0b:6e:59:87:bf:a3:ca:70:99:1b:1c:fd:72:3d:
-                    b2:d3:94:4a:cf:55:75:be:1f:40:ec:55:35:48:2d:
-                    55:f0:00:da:3c:b0:60:ba:11:32:66:54:0b:be:06:
-                    a4:5e:b7:c9:59:bb:4d:f4:92:06:26:48:6e:c2:12:
-                    d4:7c:f0:20:b8:a2:e1:bc:6a:b6:19:0e:37:47:55:
-                    c9:f2:49:0d:96:75:a2:84:64:bf:34:fc:be:b2:41:
-                    e4:f5:88:eb:e1:b7:26:a5:e5:41:c2:20:0c:f6:e2:
-                    a8:a5:e7:76:54:a5:fb:4b:80:05:7d:18:85:7a:ba:
-                    bc:b7:ad:c0:2f:60:85:cc:15:12:1c:2f:0a:9e:f3:
-                    7c:40:cf:f4:3e:23:d2:95:ca:d0:06:58:52:f0:84:
-                    d8:0f:3d:eb:ff:12:68:94:79:8f:be:40:29:5f:98:
-                    c8:90:6c:05:2f:99:8c:2a:63:78:1f:23:b1:29:c5:
-                    e7:49:c9:b2:92:0f:53:0b:d5:71:28:17:c2:19:bf:
-                    60:bf:7c:87:a8:ab:c1:f4:0a:c1:b8:d2:68:ee:c1:
-                    ce:a7:13:13:17:6d:24:5d:a2:37:a6:d7:7d:48:8b:
-                    2b:74:2d:40:2e:ca:19:d5:b6:3e:6c:42:71:fa:cf:
-                    85:87:f9:de:80:73:8b:89:f4:70:f0:d8:d7:ff:40:
-                    41:9c:c7:15:6d:9b:6e:4c:b5:52:02:99:79:32:73:
-                    ca:26:a0:ac:31:6f:c4:b0:f5:da:bb:c2:1f:e0:9f:
-                    44:ba:25:f7:9f
+            RSA Public Key: (1024 bit)
+                Modulus (1024 bit):
+                    00:b7:7f:d6:0b:d4:62:15:74:97:9b:d4:17:0a:44:
+                    07:22:0f:fe:38:05:74:b8:e3:ba:28:91:26:29:bf:
+                    70:02:ac:d9:f2:83:bc:99:a4:70:e3:3c:84:b7:5e:
+                    84:0c:b8:b8:7d:39:a9:ab:9b:b9:9e:9f:a9:2a:6b:
+                    fa:40:70:6c:32:a4:93:1a:a9:9b:07:cc:18:21:e4:
+                    09:9d:78:16:f1:d5:1e:f2:6f:4e:5f:29:7f:d5:41:
+                    ea:20:a0:1e:f7:b2:7c:d0:bf:e5:7a:fe:c5:8a:fd:
+                    e4:aa:06:56:f1:42:70:be:b9:2f:34:04:8b:d0:b6:
+                    d6:8f:a4:4c:99:25:37:c2:9d
                 Exponent: 65537 (0x10001)
-    Signature Algorithm: md5WithRSAEncryption
-        08:75:dc:b9:3f:aa:b6:7e:81:7a:39:d1:ee:ed:44:b6:ce:1b:
-        37:c4:4c:19:d0:66:e6:eb:b5:4f:2a:ef:95:58:64:21:55:01:
-        12:30:ac:8a:95:d1:06:de:29:46:a4:f1:7d:7f:b0:1e:d2:4e:
-        fb:f6:fa:9a:74:be:85:62:db:0b:82:90:58:62:c5:5f:f1:80:
-        02:9f:c5:fb:f3:6b:b0:b4:3b:04:b1:e5:53:c2:d0:00:a1:1a:
-        9d:65:60:6f:73:98:67:e0:9c:c8:12:94:79:59:bf:43:7b:f5:
-        77:c8:8f:df:b1:cd:11:1c:01:19:99:c2:22:42:f7:41:ae:b4:
-        b8:1a
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:FALSE
+            X509v3 Subject Key Identifier: 
+                CA:93:B6:F5:77:50:56:DE:5F:9E:9B:4A:39:8D:D4:8A:BD:52:8C:BE
+            X509v3 Authority Key Identifier: 
+                keyid:B1:FB:7E:77:B5:40:8D:68:11:7E:D9:11:E5:C3:C6:DA:4D:AC:51:B0
+                DirName:/C=SE/ST=Uppsala/L=Uppsala/O=MySQL AB
+                serial:BE:58:AD:4C:1F:3D:25:95
+
+    Signature Algorithm: sha1WithRSAEncryption
+        38:f0:4b:5d:f7:c6:b9:88:53:d0:d5:41:e4:c6:41:1a:1b:23:
+        14:eb:06:63:48:90:09:e9:2d:e8:51:51:c0:03:ad:61:6f:0f:
+        9e:2c:d0:3e:6d:30:ec:78:64:55:c1:e2:00:d8:2a:8d:dd:4a:
+        e7:78:22:31:8a:3d:8a:1f:34:d7:81:fb:ad:34:ac:d8:ac:de:
+        68:76:ba:ad:f3:77:70:1f:19:c7:18:9d:55:c0:ed:9f:f5:36:
+        8b:38:15:10:c0:b7:93:18:df:5a:3c:ba:9b:70:99:42:03:99:
+        da:46:e7:07:b6:c3:ea:55:a0:70:a8:2a:44:53:f1:b0:7d:b6:
+        1c:2e:80:bf:5e:da:a5:0d:4e:10:8a:76:78:11:a7:60:b3:d6:
+        5e:8f:3b:17:62:2b:8e:b3:fc:43:a1:8c:73:54:fc:b9:73:9a:
+        79:4d:bb:84:a4:fa:d1:7a:61:1d:ca:c2:49:16:f0:e8:bf:1d:
+        5c:91:41:95:5f:ac:e7:a4:63:a9:d3:1d:49:30:e3:fc:20:7d:
+        5b:07:14:94:e1:20:a6:bc:41:7b:27:82:d1:5e:2a:75:0e:74:
+        b2:bc:31:13:6d:59:8f:7b:db:25:6d:7b:64:e9:c0:7a:9e:79:
+        33:49:f6:b8:a2:ee:11:48:3a:c8:85:1e:6d:e4:10:6e:d4:8e:
+        81:7b:05:27
 -----BEGIN CERTIFICATE-----
-MIIFfDCCBOUCAxAAAzANBgkqhkiG9w0BAQQFADBEMQswCQYDVQQGEwJTRTEQMA4G
-A1UECBMHVXBwc2FsYTEQMA4GA1UEBxMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwg
-QUIwHhcNMTAwMTI5MTIwMTUzWhcNMTUwMTI4MTIwMTUzWjBDMQswCQYDVQQGEwJT
-RTEQMA4GA1UECBMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwgQUIxDzANBgNVBAMT
-BnNlcnZlcjCCBCIwDQYJKoZIhvcNAQEBBQADggQPADCCBAoCggQBAMqqHcQR7JHw
-x/9fkJL8QAxetz0AxSDVD4kxB9dBTItggKo4FN6Ta5x0iEFotQJBAS2GonqVU157
-Zy9sHilR+UT9SoC+siOhPhs4z4jEce74a0HFLcDDUqxZfYE0GZUyuJpRtkE21MSh
-roTmOLnov5a+GXprd03g3uaztmu8Pd1ovEvE6/U2k+1WohVQihDo1iLtbLHNwxjJ
-9grh3mFlYtYUQYy1+xRowc8SXUEhnVcRQ327Qywhu8NEfajPH8NxdbVHwn3OODxz
-ZJ4V2Kcnz71AyEUI48g5qAuOwlt78UeREpHM4QDglFu9MuQMjcO+zHYyUhJpsBjg
-sMJ2NFpfedn2gZ0CCmFpHDPOSfp2Ax4HWycLvzSeNJa4A5tQOmovF3oUz2VjADdS
-qHPOSxRA9NKaVlQzuHcuQluP7B8Y9K2rikqNbXAl81jny2ZRFH0W9OttVnZ2UW7W
-HdrTjcBkWmdOr+K/M9G49ir8V4enNV6Ayaz8h8lxF5G/t02j7TwbJ/RmoPlGAyfM
-6oD2S0D2QZTNvQqz7ya+3m9prg8/HFVjM5Cb7cpaEk3eSwbCopKwQj0xr6QVEhX4
-iumIjc/9hWZQbxHxn0jztbqdhmgkol2ofFRC+ti1xfLdDg/QaORUfsW5oJtlLXf0
-j7kwCtWGXO3JfNHanQ1jUO7lHpJjzKIM6EqWAk3cj998jwgYqDCI16+JrfxXSxD5
-8ctI6LY7yD/8wtPRShA8G2tk3OVlHluy2rHiJJeP7sBLjhiDfBemPEWzYAYj8i8Y
-E54XisZyeYxNBPOd6uAl0zOMHhFHYx+lRT+9hbP+pWjuSLcMpMl/ctB1Zptq+aBQ
-86hZbaPdOE9wK7v/ki5xq+/pAO0N0bRv8I6yCftNYQ3ZENVUEc0DlIT9qGjkRW4e
-ah4vhaFt9bbA8e73Nun+wvetzBNGW4hC8C0ftQ5+tSvkjau5hzBqPRL0rfMcrMwa
-SCkqlnuAAAtuWYe/o8pwmRsc/XI9stOUSs9Vdb4fQOxVNUgtVfAA2jywYLoRMmZU
-C74GpF63yVm7TfSSBiZIbsIS1HzwILii4bxqthkON0dVyfJJDZZ1ooRkvzT8vrJB
-5PWI6+G3JqXlQcIgDPbiqKXndlSl+0uABX0YhXq6vLetwC9ghcwVEhwvCp7zfEDP
-9D4j0pXK0AZYUvCE2A896/8SaJR5j75AKV+YyJBsBS+ZjCpjeB8jsSnF50nJspIP
-UwvVcSgXwhm/YL98h6irwfQKwbjSaO7BzqcTExdtJF2iN6bXfUiLK3QtQC7KGdW2
-PmxCcfrPhYf53oBzi4n0cPDY1/9AQZzHFW2bbky1UgKZeTJzyiagrDFvxLD12rvC
-H+CfRLol958CAwEAATANBgkqhkiG9w0BAQQFAAOBgQAIddy5P6q2foF6OdHu7US2
-zhs3xEwZ0Gbm67VPKu+VWGQhVQESMKyKldEG3ilGpPF9f7Ae0k779vqadL6FYtsL
-gpBYYsVf8YACn8X782uwtDsEseVTwtAAoRqdZWBvc5hn4JzIEpR5Wb9De/V3yI/f
-sc0RHAEZmcIiQvdBrrS4Gg==
+MIIDIjCCAgqgAwIBAgIBAjANBgkqhkiG9w0BAQUFADBEMQswCQYDVQQGEwJTRTEQ
+MA4GA1UECBMHVXBwc2FsYTEQMA4GA1UEBxMHVXBwc2FsYTERMA8GA1UEChMITXlT
+UUwgQUIwHhcNMTAwMTI5MDYwNDMyWhcNMzAwODEyMDYwNDMyWjBDMQswCQYDVQQG
+EwJTRTEQMA4GA1UECBMHVXBwc2FsYTERMA8GA1UEChMITXlTUUwgQUIxDzANBgNV
+BAMTBnNlcnZlcjCBnzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEAt3/WC9RiFXSX
+m9QXCkQHIg/+OAV0uOO6KJEmKb9wAqzZ8oO8maRw4zyEt16EDLi4fTmpq5u5np+p
+Kmv6QHBsMqSTGqmbB8wYIeQJnXgW8dUe8m9OXyl/1UHqIKAe97J80L/lev7Fiv3k
+qgZW8UJwvrkvNASL0LbWj6RMmSU3wp0CAwEAAaOBozCBoDAJBgNVHRMEAjAAMB0G
+A1UdDgQWBBTKk7b1d1BW3l+em0o5jdSKvVKMvjB0BgNVHSMEbTBrgBSx+353tUCN
+aBF+2RHlw8baTaxRsKFIpEYwRDELMAkGA1UEBhMCU0UxEDAOBgNVBAgTB1VwcHNh
+bGExEDAOBgNVBAcTB1VwcHNhbGExETAPBgNVBAoTCE15U1FMIEFCggkAvlitTB89
+JZUwDQYJKoZIhvcNAQEFBQADggEBADjwS133xrmIU9DVQeTGQRobIxTrBmNIkAnp
+LehRUcADrWFvD54s0D5tMOx4ZFXB4gDYKo3dSud4IjGKPYofNNeB+600rNis3mh2
+uq3zd3AfGccYnVXA7Z/1Nos4FRDAt5MY31o8uptwmUIDmdpG5we2w+pVoHCoKkRT
+8bB9thwugL9e2qUNThCKdngRp2Cz1l6POxdiK46z/EOhjHNU/LlzmnlNu4Sk+tF6
+YR3KwkkW8Oi/HVyRQZVfrOekY6nTHUkw4/wgfVsHFJThIKa8QXsngtFeKnUOdLK8
+MRNtWY972yVte2TpwHqeeTNJ9rii7hFIOsiFHm3kEG7UjoF7BSc=
 -----END CERTIFICATE-----
diff -Naur mysql-5.0.95.orig/mysql-test/std_data/server8k-key.pem mysql-5.0.95/mysql-test/std_data/server8k-key.pem
--- mysql-5.0.95.orig/mysql-test/std_data/server8k-key.pem	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/std_data/server8k-key.pem	2012-02-03 13:45:19.559949550 -0500
@@ -1,99 +1,15 @@
 -----BEGIN RSA PRIVATE KEY-----
-MIISKgIBAAKCBAEAyqodxBHskfDH/1+QkvxADF63PQDFINUPiTEH10FMi2CAqjgU
-3pNrnHSIQWi1AkEBLYaiepVTXntnL2weKVH5RP1KgL6yI6E+GzjPiMRx7vhrQcUt
-wMNSrFl9gTQZlTK4mlG2QTbUxKGuhOY4uei/lr4Zemt3TeDe5rO2a7w93Wi8S8Tr
-9TaT7VaiFVCKEOjWIu1ssc3DGMn2CuHeYWVi1hRBjLX7FGjBzxJdQSGdVxFDfbtD
-LCG7w0R9qM8fw3F1tUfCfc44PHNknhXYpyfPvUDIRQjjyDmoC47CW3vxR5ESkczh
-AOCUW70y5AyNw77MdjJSEmmwGOCwwnY0Wl952faBnQIKYWkcM85J+nYDHgdbJwu/
-NJ40lrgDm1A6ai8XehTPZWMAN1Koc85LFED00ppWVDO4dy5CW4/sHxj0rauKSo1t
-cCXzWOfLZlEUfRb0621WdnZRbtYd2tONwGRaZ06v4r8z0bj2KvxXh6c1XoDJrPyH
-yXEXkb+3TaPtPBsn9Gag+UYDJ8zqgPZLQPZBlM29CrPvJr7eb2muDz8cVWMzkJvt
-yloSTd5LBsKikrBCPTGvpBUSFfiK6YiNz/2FZlBvEfGfSPO1up2GaCSiXah8VEL6
-2LXF8t0OD9Bo5FR+xbmgm2Utd/SPuTAK1YZc7cl80dqdDWNQ7uUekmPMogzoSpYC
-TdyP33yPCBioMIjXr4mt/FdLEPnxy0jotjvIP/zC09FKEDwba2Tc5WUeW7LaseIk
-l4/uwEuOGIN8F6Y8RbNgBiPyLxgTnheKxnJ5jE0E853q4CXTM4weEUdjH6VFP72F
-s/6laO5ItwykyX9y0HVmm2r5oFDzqFlto904T3Aru/+SLnGr7+kA7Q3RtG/wjrIJ
-+01hDdkQ1VQRzQOUhP2oaORFbh5qHi+FoW31tsDx7vc26f7C963ME0ZbiELwLR+1
-Dn61K+SNq7mHMGo9EvSt8xyszBpIKSqWe4AAC25Zh7+jynCZGxz9cj2y05RKz1V1
-vh9A7FU1SC1V8ADaPLBguhEyZlQLvgakXrfJWbtN9JIGJkhuwhLUfPAguKLhvGq2
-GQ43R1XJ8kkNlnWihGS/NPy+skHk9Yjr4bcmpeVBwiAM9uKoped2VKX7S4AFfRiF
-erq8t63AL2CFzBUSHC8KnvN8QM/0PiPSlcrQBlhS8ITYDz3r/xJolHmPvkApX5jI
-kGwFL5mMKmN4HyOxKcXnScmykg9TC9VxKBfCGb9gv3yHqKvB9ArBuNJo7sHOpxMT
-F20kXaI3ptd9SIsrdC1ALsoZ1bY+bEJx+s+Fh/negHOLifRw8NjX/0BBnMcVbZtu
-TLVSApl5MnPKJqCsMW/EsPXau8If4J9EuiX3nwIDAQABAoIEAElnTjqq502AsV+c
-hGfId4ZDdAjjU4LtyJ+/I4DihM/ilxeQEnb/XDWhu4w9WXpEgyGzJvxRQ43wElKJ
-zW7X4voK58Yzy5++EhmX/QsjY8TTMz3yJf0wgawtCZkXfsCcS2KRf/qk2nGRwf0e
-yaMEWwhFOEMv01lgvjs/Ei55Usrz2Wd0HqaFKxUGkNQ5hJhVTOH/rqPDzAsZc0VD
-w+Dw8NhrI8bMTvF4c+IFW8NwYmWbuh87CTxdx30VPJI82ttWJ/UN1bLtU08J2IKt
-lPgOIl8ArMjcTGxD/cqZ3Wl3Pc/XCqvGUiSYMwP7Rgh1R4+DdtjEpxdGMmMAVuVI
-HPQyqpa4gv+UMqBPish0yjSuM7jXnztINOvg9Vk1sxC5AT9eaRltmiS1s+lVxe+T
-43ulf0ccYXJD/WclWSGCwloNFuokPIV+Lgo1pKsp4XDgoxQfkXwH8Q4dEqebY9rT
-Tv9FGb1bMbdl22X1oSu2lBltBZaB/QnruV7L2GaQ0tqLKizgBRuvZFSE+DWdMb6d
-9mnEB8LWtca/nzogXb5qv4GEMUX4FUAmSf1FnGWZwwDi1DFfJ860RVKf0xokGGQ3
-cm3H/F4veds88Z1hsAu0bG8h/bEAim+Whvag995cFHDD4on41KXW8wX1on9VFA1W
-CkaGUPhLRytXDBVCSJkOYYFSJlb2wqONiWe4Tn5hsantCfliTj/GVkgDq2h7dAGR
-WyoqTntJAv/xJsUOV9WmGXnWNeZX8BSO3P5dnXnMzhCWQGoprXmWFyJ3TYCJ2+CO
-rzkZbtuKvTvGc3sDJgrSVmmg0BrOkH+GyYVlJdTDBmfzoORludDCFHECa8oK7NwY
-t3o0eNlG6IqTxl2HIoPneW9nXFQtCXv6tpJjljwjlz5WpJG+kBW6bDedcxZu7olZ
-fqtnyZTB2SjzzbGdQ4JvFup8MxNyPvYiqumQXJgkyXFVDl/UFhjWuGe04i8NBJgJ
-xORcjfgLrKH1XKVBWPJdh/2YeUKIIvQ9RB4WVqXgGmD/21tgv1bVEMYabh23e/HE
-Fe1U2XQPJKxGCEtG6b4zhFP+PeZACS+Vk5IVJYK9n4SepPBPgX/wbJLOcKGpsKjp
-yx5WjopMO6T+VUV8HIduuZ+E8+uAILHDmo2Bq+LHblaxd4SkM0+hL2H36imK5CUO
-5fLuvHW88LvFtQw6xhP20s+BnmgzE5ZvNG4Iedkjvwe9HmdNDew0UYT5vNJN0ehh
-OlraBC++JYwEclrBD9SRvprT63XKDG735pPvzLQi7WKDCBn1/JEgxDIO8nkMewOZ
-FU48Mdmkn9wqPeIigQciwl62fuAQCGRG+RXMQqra4A1apqMZQEauTK50VhHDGdbc
-ye9LHaECggIBAO9lAzoYS/Lu0ticMt24P8BSbGdxSNIpEyIlTTs+7A0UjpfXsoK9
-4EJWZ7lhgbQh+SCTS662SeC+s8M6bT+3mELxUC5S/N3aCPyfjcM3JaoACkI9+VMn
-9otJZjAEwH7cNpMN0Xa8fHCEma3l3XKiVxEJbuJC86S5mpkjeXVnDajAidBtevBd
-LWJ9n2yXk+ZKUyI0mjpqItwUxOgQ/MOIvqAu66xyjg08/I1QQTuIrReAA+oaVKhp
-c42Ufn26hUhNrQCBAtMAO3VC/chciet6vEMNEM13GqLp4+PcPhRX90gO4+bNrScD
-WgiW/jc24CGan8gAenBWC/3l/C6JUsMp+ZYmPozsa0zo6edgiO/f2KXe5nP87wZT
-MxaYJgnyXJxMefI79kUHPrhpXZxuiSCEWLhCBN34Lhpr2L491i2g/FJj9i6N3EzE
-N3ic5Q63o4QFusjqIm3taQQFoGP2Cgg9owz5WJ0uRz/gtOE3XQiQA7+ozoAXOlTw
-pJK5MMtVrEoOLIbVJIpxfDcKDp3yorR8QCQLHgDBmFeNCDmk+7YP33dRIc/AVNLF
-q7cecqEc7D8AkXX8Q53GfCEg+uqbdeMQXK4BUE9iwRK9RiFhas/RJe73+Iio3S0L
-ekLpnnOfvk744ws+JWsLpsfC/ZE7OxBLPtq2xvGl/RT2G7tCjmpX3CbPAoICAQDY
-uOEJks2T105EcMPJjzNHCCqjK6S7qZaWkF3KT1Z0Mu5oUZwwHamsMg4BQJ2mjMrL
-fRBKfXQLA6vgE7zysw3F300RDxE1RVow5+JLDQ4bqupp27/M0a8fuwksyOdKHqCV
-YHzuTCxbVIFZawTjfOxJVXDHKCFCilfY1LsA+V+oFe3Ej8YYxWXkXA9ZLigpmt3s
-Wu6eFcZgF3utzIGjI6eP6lL5bWp6Bh9Avp2xrOvpFwE2m02Y7/Zom6MT4DXvByY2
-KHHQLsasEMpeLuxQXjLeTocwcxBwBFKhX95yFuv31k00VydT+NExtaZeUYi9l19J
-WmM4GjFjAqa3uUwMNVv5JfWtKMyk4FOox2XftLvMiIhV95B8hAGxtYr3hPkGg80O
-AWPq6OKUD332COXRaHkmL5aQdN3gP5zh9+rH6icLrrZbrQidVRyDw03doRoGrH7i
-ixXLyYoW80PHgqUDPohd5bFkZpi2vwXMl1YQ2TfN9TvYFSGme9YCm9ZuypnqauW/
-aAf0FI1MNwS+XDREtzPdFi0me6WxpKL4a2Z3GGNxIFuBjQ/uydWpjxkny9qI3KAp
-SgjI3kBUDGq3gf0R+Xo/d4d/4asK9Nv2Fi0X+RfGqioFaTbQl/1zhNdvhP9IcwEJ
-DLVQ3UhMdfg285RarC2Sihui0M8Smi9od9Dj6rdWMQKCAgEAiQVRFoRnnDGz/wVQ
-W/Wkj6jdoUuG+btG10lwbhOyuj3k6+Yqp4iUfoPENKgpu/eiB1InhGWT3Y5ph7m+
-ZDTqco56bTlUwIqWkDmmw3CiHy6MsKOWPFFoXQry8VMW9sWGex7yoDp8I07SQ2WJ
-HZ7rpLW4gMr/d25AnZxfXaJRgCBMAT9YmZFLc88hW99aaPproO1oxTyQnVVJ6uYm
-NqjjKv4QKJEc21jn2N5xp+iv4f6Evw65G/fXitbOm5oRxXOoLNyqyCie35wrc+37
-hwumC97DmkasuUiUBoy9/5jl0ZmsOiPJEsZpVvdNpD7FhJZjE++qJPgrPvTPJbe1
-5jz1PUrAjJqZQ9kgYC2x01JVR4NQdlz0VrNyT2FgjFrrRQ7E0bAeYh4meRjd2rat
-yC3YNgabkI0HnlnSIfl0yIMXSPUsKDNMP6gjc+aheI4FioBZC7xvXmn/rKynw+9E
-iLj2xWtGnBir8VTlUu8EUe1UJ/Qv1cL1wT5HhC95TTjJN03rkHUYyCDyjvIzsZX6
-KMHhWIAAeUBVuO7hIVVcOTXWmw2WA7o7ErTPdy13QN40Hk9t8pEkBn9f9vpQg83d
-aMypr3LTC80jY11wcZS3tSEpzCCkYVv91FV4cioTZmytWbg9A+dbNWzi1f22ctTr
-FoVrAXaSYie2trOy5bjPmPCW8qMCggIBALQUKymBSkDmTqqf6I+65ajIKGWdBizJ
-Jc/F9aj9c6DqER+tcFKq0ym6DdkMj/KsWnXrXXYH+DyOuGpg/EfOcEtS2P6rvmi9
-T8wDYg1qs6ZZxp5fcmgGc7Wx/FWyOj1kZZq5qhV4RgM9nJ1oR4+fZdcpn6RcvAZG
-XehWG20byVgpoIAL11cN7zRpKne32rd3b5/NjyjcfxGpcaNgovej0L/MvVV0jV0H
-aUCrIu1X+k6cRu3Q7hF+kwkpCcCiNS6AikfGI4wQ0hR3fy/zXXkKTMpcBglEEwyB
-Cwf8WSID2d79uvka0hr8TRc5ERyeMzkWZp7U9EzRtufGdDGFTqN2Uw4bdKCFnkYC
-AIHl7ciMrN+vM1n7c5uDNMUtTGOPojy/l8tjbFrtWBgfJ1Mg4ZW3cbNBJ6Kw+Qw0
-z28USYoEDp2uduiGRvo0lpUF29Wk37Nb8bLcTygeNxgK2u8Up3iipT0gdt4uQgbX
-g0IVHfayB6SjeS57oJJto85XHz7AKlSWroD1OGagDSifLtneU7AlanryymGHrI6H
-dsNkuqeLJFYDxQVI6UxJebiCpyxiPxwp9wtX8SS3SEyOZL5GzLn6ypGiCH1CTpW0
-EHHSy3V4DUGOc4w7eMirAnbSkxCfOmBA70NNw/uFY2XlQHKow0T0fImfKIeJagbT
-B0GPDYvUpLKBAoICAQCzYnq8xupXK7lvTLaj936qGSe54OC2sj9+UpsFiPxglNY2
-sO5zKWKyY7+rjK6zG2ciGfPEDsZNIqKw1W/KBfR2kRLqkt4bC3fSCvUztx0vtGUe
-veXlqiwETdE7RJXoaGJrgJArYJvpOd8PtWGeM+sSJNNrUlGlJnSiZ0CcypqUZgZL
-WzGFfLOQYAXCykdB1iZkBqU2C5wktvCb9sVz6G3TmAwSKTENOWWZWmh+W0J4pZFV
-ZEyvsxViJRQbwxa0kC0F5J/UtWZknO79/ZFj1H4jiAR45EjWHE+UZAkFwG8BSl54
-EKOx7GDanuRILr0dtbyi4d31nCYXdjs3x2+1N3exw4oKQIvNuF54WoowbNPu0kEb
-G+7/kLwcJqRnSV4AiLuMz5aOte7JJSw5tzgZZlAQwJO7IDfrLqodivcXF5yirwiF
-dyBpzSDmupy/aTHnCpT+l0H96jRU2awxaeRHZUqZog8gMHsslNVZEFvUFDJ7AUN/
-yyfUzJYjH18pZt0hS7jNb1O7KxZCkWGMiEcxHkgF/UINab5qruNBVKOkJ5vqGhYi
-uNkgeGsQtXJcpqMRRiVXJE0kE+26gk+iaYnBJN9jnwy8OEAlYFUHsbCPObe/vPMQ
-3RLl+ZoKdFkN/gTiy70wUTRVw+tWk+iAZc7GPX1CqDFOqGZ2t+xdF8hpsMtEww==
+MIICXgIBAAKBgQC3f9YL1GIVdJeb1BcKRAciD/44BXS447ookSYpv3ACrNnyg7yZ
+pHDjPIS3XoQMuLh9Oamrm7men6kqa/pAcGwypJMaqZsHzBgh5AmdeBbx1R7yb05f
+KX/VQeogoB73snzQv+V6/sWK/eSqBlbxQnC+uS80BIvQttaPpEyZJTfCnQIDAQAB
+AoGAIKRC5TPQhtSIq5Wcy5yyUCLcLAb8bm9EDDdVvJNrT26EDpnUkNgppKJ9UUkk
+4EZzuwLo5bf72vW31+4BRDk7ZbGwgPGtrFtSyQ4oNKvaeGAQguPl+XOO3qSnrP49
+H6nJhl92CvOlKJfk8voCvA8yk2cAmlZQKWp9w/6pl/R0n8ECQQDfEkZ3VO9dYTMX
+098zJuxvs4ClTNopdkGyWSn3DgHil1MAwxwEfEjr7MX8Ilfj1XtqSvWzZfTMkcyA
+aTPCkHnlAkEA0pYox3NrG9wJFYafZLWay2wrjf6XXkdz8zXAuwa6uNAv8hIGOb7o
++w2RO5N2D4ofIPPv0JaD+evnoH5ey+m6WQJBALvpIjEN/GGWKrsswH3PkSZ7eKhG
+uBRrSOZbVA2I4C+nt8URlr10kN0k5JcDuzPytvcQVC+eeHwHadTWSkLg/70CQQC4
+aidsAvdZfme2Ih6T4zWbzy6VOs70ET7JTdlp74q1LhvOsVoiV71PnaIBujE5iouY
+IAdzc1xAEQbW1dPOXvCBAkEAreTSucJFznJ6TtQcXIdxJUt883apFQyJDeaHIzkN
+QXYuDGZmxRwW/9Q/jACU0gu4fj3Q3OuzdeIsMW/3NoEb+g==
 -----END RSA PRIVATE KEY-----
diff -Naur mysql-5.0.95.orig/mysql-test/t/openssl_1.test mysql-5.0.95/mysql-test/t/openssl_1.test
--- mysql-5.0.95.orig/mysql-test/t/openssl_1.test	2011-12-16 07:22:48.000000000 -0500
+++ mysql-5.0.95/mysql-test/t/openssl_1.test	2012-02-03 13:50:17.114564324 -0500
@@ -15,8 +15,8 @@
 
 grant select on test.* to ssl_user1@localhost require SSL;
 grant select on test.* to ssl_user2@localhost require cipher "DHE-RSA-AES256-SHA";
-grant select on test.* to ssl_user3@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB";
-grant select on test.* to ssl_user4@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB" ISSUER "/C=SE/ST=Uppsala/L=Uppsala/O=MySQL AB";
+grant select on test.* to ssl_user3@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB/emailAddress=abstract.mysql.developer@mysql.com";
+grant select on test.* to ssl_user4@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "/C=SE/ST=Uppsala/O=MySQL AB/emailAddress=abstract.mysql.developer@mysql.com" ISSUER "/C=SE/ST=Uppsala/L=Uppsala/O=MySQL AB";
 grant select on test.* to ssl_user5@localhost require cipher "DHE-RSA-AES256-SHA" AND SUBJECT "xxx";
 flush privileges;
 
